src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -31,18 +32,34 @@ const initialState = {
     email: '',
     firstName: '',
     lastName: '',
+    hasEnrollment: false,
```

Comment:
You confirm that by default the instructor doesn't have permission to enroll?

Reply:
And please change the name to the variable to hasEnrollmentPrivilege to have consistency with instructor portal